cmd/builds.go

```diff
@@ -35,13 +35,22 @@ func (d *DeisCmd) BuildsList(appID string, results int) error {
 }
 
 // BuildsCreate creates a build for an app.
-func (d *DeisCmd) BuildsCreate(appID, image, procfile string) error {
+func (d *DeisCmd) BuildsCreate(appID, image, procfile string, deploy_now bool) error {
 	s, appID, err := load(d.ConfigFile, appID)
 
 	if err != nil {
 		return err
 	}
 
+	config, err := config.List(s.Client, appID)
+	if d.checkAPICompatibility(s.Client, err) != nil {
+		return err
+	}
+
+	if val, ok := config["DEIS_DEPLOY_NOW"]; ok {
```

Comment:
This looks good but haven't tested locally -- would be nice to have this covered by an added unit test  in `builds_test.go` as well (in fact, I don't see any tests yet covering scenarios with `deploy_now` being `false`...)